src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/model/Availability.kt

```diff
@@ -9,4 +9,6 @@ data class Availability(
   val nonArrivedBookings: Int,
   val cancelledBookings: Int,
   val lostBeds: Int
-)
+) {
+  fun getFreeCapacity(totalBeds: Int) = totalBeds - pendingBookings - arrivedBookings - lostBeds
```

Comment:
Is it worth bracketing this to be clearer?

```suggestion
  fun getFreeCapacity(totalBeds: Int) = ((totalBeds - pendingBookings) - arrivedBookings) - lostBeds
```

Reply:
Have done